scripts/linters/pre_commit_linter.py

```diff
@@ -32,17 +32,17 @@
 2.  To lint all files in the folder or to lint just a specific file
         python -m scripts.linters.pre_commit_linter --path filepath
 
-3.  To lint a specific list of files (*.js/*.py only). Separate files by spaces
+3.  To lint a specific list of files. Separate files by spaces
         python -m scripts.linters.pre_commit_linter
             --files file_1 file_2 ... file_n
 
 4.  To lint files in verbose mode
         python -m scripts.linters.pre_commit_linter --verbose
 
-5. To lint a specific list of file extensions (*.js/*.py only). Separate file
+5. To lint a specific list of file extensions. Separate file
     extensions by spaces
         python -m scripts.linters.pre_commit_linter
-            --only-check-file-extensions .py .js
+            --only-check-file-extensions "*.py" "*.js"
```

Comment:
Hm. If you need the quotes then that starts becoming kind of annoying to type. In that case, maybe let's just do `py js` then. (Sorry!)

Reply:
Done.